sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1073,15 +1073,34 @@ object functions {
   def floor(columnName: String): Column = floor(Column(columnName))
 
   /**
-   * Computes hex value of the given column
+   * Returns the greatest value of the list of values.
    *
-   * @group math_funcs
+   * @group normal_funcs
    * @since 1.5.0
    */
+  @scala.annotation.varargs
+  def greatest(exprs: Column*): Column = Greatest(exprs.map(_.expr): _*)
```

Comment:
Should we check that `exprs` have at least two columns? Also for others, it's good to fail fast.